### ISSUE TITLE
Support to fetch file that set express's router automatically.

### ISSUE THREAD

**Issue by foway0** [cdimascio/express-openapi-validator#177]
I'm writing an issue through Google Translator. I hope the contents are well communicated.

Sir,
Your project is amazing.
However, it would be nice to have additional features.
I'd like to propose to consider introducing additional features, so I'm writing an issue.
If you don't mind me asking please to consider the following features.
For example, set `operationId`, `eov-controller` (name is just an example) in `api.yaml` as shown below.
I think it would be cool if the routing configuration is automated.

I hope to hear from you soon.
Thank you. I wish you good luck.

- api.yaml

```yaml
openapi: '3.0.0'
info:
  version: 1.0.0
  title: Swagger Petstore
  description: A sample API
  termsOfService: http://swagger.io/terms/
  license:
    name: Apache 2.0
    url: https://www.apache.org/licenses/LICENSE-2.0.html
servers:
  - url: /v1
paths:
  /pets:
    get:
      description: |
        Returns all pets
      operationId: 'pets#findPets' # will be automatically fetched
      eov-controller: 'pets' # load pets.js
```

- pets.js

```javascript
module.exports = {
  'pets#findById': (req, res, next) => {
    res.json(pets.findAll(req.query));
  }
};
```

- don't need to write code like this

```javascript
// It will be fetched automatically.
app.get('/v1/pets', (req, res, next) => {
 res.json(pets.findAll(req.query));
});
```

**Comment by cdimascio** [comment]
Thanks @foway0. This is a nice idea. It's certainly worth considering. 

We do welcome PRs.

If you would like to attempt an initial pass at implementing your proposed feature, I will certainly consider integrating it. 

Let me know if you would like to give it a try. 

If so, there is also a gitter channel that we can use to chat in real time if desired. https://gitter.im/cdimascio-oss/community

**Comment by foway0** [comment]
@cdimascio 
Thank you for reply.

Okay, I will try it once.
Please let me know if there are any procedures for PRs.

**Comment by cdimascio** [comment]
Fantastic! 

There is no special procedure for PR. Essentially, implement the feature, create tests, and ensure existing tests continue to pass.

I use prettier for linting/formatting. Codacy will check code quality on push and make suggestions. Use the recommendations that make sense. It’s not mandatory to conform to all of its suggestions.

See [CONTRIBUTING.md](https://github.com/cdimascio/express-openapi-validator/blob/master/CONTRIBUTING.md) to get started

if you have questions, feel free to reach out here or on [gitter](https://gitter.im/cdimascio-oss/community)

I very much appreciate your help. Thanks!

**Comment by nagarajanceg** [comment]
Hi Cdimascio,
    This feature is really cool and reduce additional boilerplate code to set up express router. Also provide ease in debugging by looking at yaml to figure out the controller. It would be great if this feature is available.

**Comment by foway0** [comment]
@cdimascio 

Sorry for I am late. I'm learning typescript first time.

I create a new branch. (WIP yet) 
[https://github.com/foway0/express-openapi-validator/pull/2/files](url)

When initializing OpenApiValidator, you can specify a controller as an option. If you set an absolute path in the controller,  routing is automatically initialized by interpreting the `x-eov-operationId` and `x-eov-controller` in the api-spec definition.

```
x-eov-operationId: the key of the exported object
x-eov-controller: file name
```

※ The example code has been slightly modified for development debugging.

Please let me know if you think there is a problem.

thanks.

**Comment by cdimascio** [comment]
@foway0 Excellent! Thank you.
I will have at your work within the next day or so.
Please create a pull request in this repo. It will make it easier for me to follow along and comment.
Thank you again. I'm really looking forward to this feature.

**Comment by cdimascio** [comment]
@foway0 I've created a PR that contains your feature work
https://github.com/cdimascio/express-openapi-validator/pull/228

**Comment by nagarajanceg** [comment]
Awesome! Thanks to @foway0. It's really helpful

@cdimascio 
Are these changes are readily available in any existing version or it's going to available in future versions?

**Comment by cdimascio** [comment]
@nagarajanceg this will be in a version soon. i haven't yet gotten to review / test the new functionality. i'm hoping to get to it this weekend. 
if you're up for it, feel free to provide review comments and/or clone the PR branch #228, test it out, and raise any gaps, bugs, etc that need attention. thanks!

**Comment by foway0** [comment]
> @foway0 I've created a PR that contains your feature work
> #228

**Comment by cdimascio** [comment]
@foway0 @nagarajanceg this feature is now available in version `3.7.0-alpha.1`. please give it a try and provide feedback.
Thanks so much for your work, @foway0!
